config/settings.yml

```diff
@@ -1446,6 +1446,49 @@ mobile_lighthouse_letters:
   api_scopes:
     - "letters.read"
 
+mobile_lighthouse_benefits_claims:
+  client_id: '0oal917lvbYQbkaQu2p7'
+  key_path: ~
+  aud_claim_url: "https://deptva-eval.okta.com/oauth2/ausdg7guis2TYDlFe2p7/v1/token"
+  access_token_url: 'https://sandbox-api.va.gov/oauth2/claims/system/v1/token'
+  api_url: 'https://sandbox-api.va.gov/services/claims/v1/'
+  api_scopes:
+    - "claim.read"
+    - "claim.write"
+
+mobile_lighthouse_letters_2:
```

Comment:
Currently just calling this _2 because we have this set up with another key. The older version uses a different aud_claim_url and token url, but I suspect these may not be meaningful differences. The plan is to switch over to have all mobile lighthouse apis use the same keys.